packages/@aws-cdk/aws-cognito/lib/user-pool-idps/base.ts

```diff
@@ -36,6 +36,25 @@ export class ProviderAttribute {
   /** The locale attribute provided by Facebook */
   public static readonly FACEBOOK_LOCALE = new ProviderAttribute('locale');
 
+  /** The name attribute provided by Google */
+  public static readonly GOOGLE_NAMES = new ProviderAttribute('names');
+  /** The gender attribute provided by Google */
+  public static readonly GOOGLE_GENDER = new ProviderAttribute('gender');
+  /** The birthday attribute provided by Google */
+  public static readonly GOOGLE_BIRTHDAY = new ProviderAttribute('birthdays');
+  /** The birthday attribute provided by Google */
+  public static readonly GOOGLE_PHONE_NUMBER = new ProviderAttribute('phoneNumbers');
```

Comment:
Use plural here as well.

```suggestion
  /** The birthdays attribute provided by Google */
  public static readonly GOOGLE_BIRTHDAYS = new ProviderAttribute('birthdays');
  /** The phone numbers attribute provided by Google */
  public static readonly GOOGLE_PHONE_NUMBERS = new ProviderAttribute('phoneNumbers');
```